README.md

```diff
@@ -1,4 +1,4 @@
-# Dependency Injection Graph
+# dig - A dependency injection framework for Go
```

Comment:
This change sort of makes it not an acronym anymore.

It's up to you though, I thought the lowercase looks best. Similar thing happens to alot of acronym based names, same as yarpc which now shows as: https://github.com/yarpc/yarpc/blob/master/assets/yarpc.png